indico/modules/rb/client/js/store.js

```diff
@@ -5,26 +5,25 @@
 // modify it under the terms of the MIT License; see the
 // LICENSE file for more details.
 
-import {createBrowserHistory} from 'history';
 import {queryStringMiddleware} from 'redux-router-querystring';
 import {routerMiddleware} from 'connected-react-router';
 import createReduxStore from 'indico/utils/redux';
 
 import getReducers from './reducers';
-import {queryStringReducer as qsRoomSearchReducer} from './common/roomSearch';
-import {routeConfig as roomListRouteConfig} from './modules/roomList';
+import {queryStringReducer as qsRoomSearchReducer} from './common/roomSearch/queryString';
+import {routeConfig as roomListRouteConfig} from './modules/roomList/queryString';
 import {
   routeConfig as bookRoomRouteConfig,
   queryStringReducer as qsBookRoomReducer,
-} from './modules/bookRoom';
+} from './modules/bookRoom/queryString';
 import {
   routeConfig as calendarRouteConfig,
   queryStringReducer as qsCalendarReducer,
-} from './modules/calendar';
+} from './modules/calendar/queryString';
 import {
   routeConfig as blockingsRouteConfig,
   queryStringReducer as qsBlockingsReducer,
-} from './modules/blockings';
+} from './modules/blockings/queryString';
```

Comment:
I guess you did this to avoid importing the `index.js` directly. Have you checked whether in these cases you still need to re-export `routeConfig` and `queryStringReducer` in it? I mean [this line](https://github.com/indico/indico/blob/master/indico/modules/rb/client/js/modules/blockings/index.js#L16). If we don't need it, we're better off removing as it may be misleading for future code.